csharp/ql/lib/ext/System.Data.model.yml

```diff
@@ -54,6 +54,17 @@ extensions:
       - ["System.Data", "EnumerableRowCollectionExtensions", False, "Where<TRow>", "(System.Data.EnumerableRowCollection<TRow>,System.Func<TRow,System.Boolean>)", "", "Argument[0].Element", "ReturnValue.Element", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "get_Item", "(System.String)", "", "Argument[this].Element", "ReturnValue", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "set_Item", "(System.String,System.Object)", "", "Argument[1]", "Argument[this].Element", "value", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetDataTypeName", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetDateTime", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
```

Comment:
This API returns a `DateTime`, do we need to consider this API in taint tracking?

Reply:
Generally `DateTime` has started to be considered a more "simple" type (a form of santizer) for most queries, so we should definitely exclude such summaries from the model generation in general! I will modify the generator and re-generate (thank you for pointing this out!!!).